packages/testsuite/cypress/support/commands.ts

```diff
@@ -331,6 +344,31 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add(
+  "verifySubAttribute",
+  (
+    managementEndpoint,
+    address,
+    attributeName,
+    subAttributeName,
+    expectedValue
+  ) => {
+    cy.task("execute:cli", {
+      managementApi: managementEndpoint + "/management",
+      operation: "read-attribute",
+      address: address,
+      name: attributeName,
+    }).then((result) => {
+      expect((result as { outcome: string }).outcome).to.equal("success");
+      let str: { [subAttributeName: string]: any } = (
+        result as { result: { subAttributeName: string } }
+      ).result;
+      let abc = str[subAttributeName];
```

Comment:
please rename `abc` variable to something more meaningful

Reply:
This new function isn't necessary.